### DISCUSSION TITLE
Quit to tray?

### DISCUSSION THREAD

**Discussion by agoodfellow123** [zaps166/QMPlay2#697] [Ideas]
Hello, I'm absulutely loving this program can tray be added? what I have in mind is quite simple, for example radio stations just show this tab when the tray icon is pressed with right mouse button:

![image](https://github.com/zaps166/QMPlay2/assets/36235265/83433ec9-4b0c-4687-88dc-0e9c1df062c4)

When pressed with left mouse button show a little options menu for getting the app in it's original form.

**Comment by zaps166** [comment]
Tray is supported, also "My radio stations" are accessible from tray menu.

**Reply by agoodfellow123** [reply]
"Show tray" check does nothing, perhaps its a bug then?

**Reply by zaps166** [reply]
Where do you use QMPlay2 (OS, platform) and from where you installed it?

**Reply by agoodfellow123** [reply]
Archlinux, Flatpak. But I realized it's an outdated package. So now I downloaded the appimage works perfectly!

**Reply by zaps166** [reply]
Ok!
However on ArchLinux you can use AUR for up-to-date packages! I don't manage flatpak repo, so don't know when they update it.
